modules/mri_violations/test/mri_violationsTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Mri_violations automated integration tests
+ *
+ * PHP Version 5
+ *
+ * @category Test
+ * @package  Loris
+ * @author   Ted Strauss <ted.strauss@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+
+require_once __DIR__ . "/../../../test/integrationtests/LorisIntegrationTest.class.inc";
+class MriViolationsTestIntegrationTest extends LorisIntegrationTest
+{
+    protected $url = 'http://localhost/main.php?test_name=mri_violations';
+
+    /**
+     * Tests that, when loading the Mri_violations module, some
+     * text appears in the body.
+     *
+     * @return void
+     */
+    function testMriViolationsDoespageLoad()
```

Comment:
Can you add a test to make sure the 2 submenus load:

?test_name=mri_violations&submenu=mri_protocol_violations

and

?test_name=mri_violations&submenu=mri_protocol_check_violations